src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ConnectionPoint/ConnectionPointContainer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.OLE.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.ConnectionPoint
+{
+    /// <summary>
+    /// This implementation is a copy from CPS
+    /// </summary>
+    [ComVisible(true)]
+    public class ConnectionPointContainer : IConnectionPointContainer
```

Comment:
I forget what we did in Roslyn - I think we just moved all such types to a different namespace. But agree taht creating a separate dll and not including that in the nupkg is better. Can you file a bug?

---
In reply to: [111332078](https://github.com/dotnet/project-system/pull/1911#discussion_r111332078) [](ancestors = 111332078)